src/test/test-udev.c

```diff
@@ -61,14 +61,17 @@ int main(int argc, char *argv[]) {
 
         test_setup_logging(LOG_INFO);
 
-        if (argc != 3) {
-                log_error("This program needs two arguments, %d given", argc - 1);
+        if (argc != 2 && argc != 3) {
```

Comment:
`!IN_SET(argc, 2, 3)` can be used here.

Reply:
I've had a love-hate relationship with `IN_SET` since https://github.com/systemd/systemd/pull/10374 :-), but in this case I agree that it's better. I'll change it.

Reply:
Done.